plutus-core/untyped-plutus-core/test/Evaluation/Golden/case4.plc.golden

```diff
@@ -0,0 +1 @@
+(Right (con string "hello"))
```

Comment:
It's super hard to review these. We should pretty-print the programs alongside what they evaluate/typecheck to.